script/deploy/mainnet/005_RegisterLidoWithdrawalsScript.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.23;
+
+import "forge-std/console.sol";
+import {Vm} from "forge-std/Vm.sol";
+
+import {IERC20, IWETH, LegacyAMM} from "contracts/Interfaces.sol";
+import {LidoARM} from "contracts/LidoARM.sol";
+import {CapManager} from "contracts/CapManager.sol";
+import {Proxy} from "contracts/Proxy.sol";
+import {ZapperLidoARM} from "contracts/ZapperLidoARM.sol";
+import {Mainnet} from "contracts/utils/Addresses.sol";
+import {GovProposal, GovSixHelper} from "contracts/utils/GovSixHelper.sol";
+import {AbstractDeployScript} from "../AbstractDeployScript.sol";
+
+contract RegisterLidoWithdrawalsScript is AbstractDeployScript {
+    using GovSixHelper for GovProposal;
+
+    GovProposal public govProposal;
+
+    string public constant override DEPLOY_NAME = "005_RegisterLidoWithdrawalsScript";
+    bool public constant override proposalExecuted = false;
+
+    Proxy lidoARMProxy;
+    Proxy capManProxy;
+    LidoARM lidoARMImpl;
+    LidoARM lidoARM;
+    CapManager capManager;
+    ZapperLidoARM zapper;
+
+    function _execute() internal override {
+        console.log("Deploy:", DEPLOY_NAME);
+        console.log("------------");
+
+        // 1. Deploy new Lido ARM implementation
+        uint256 claimDelay = tenderlyTestnet ? 1 minutes : 10 minutes;
+        lidoARMImpl = new LidoARM(Mainnet.STETH, Mainnet.WETH, Mainnet.LIDO_WITHDRAWAL, claimDelay);
+        _recordDeploy("LIDO_ARM_IMPL", address(lidoARMImpl));
+
+        console.log("Finished deploying", DEPLOY_NAME);
+    }
+
+    function _buildGovernanceProposal() internal override {
+        govProposal.setDescription("Upgrade Lido ARM and register Lido withdrawal requests");
+
+        bytes memory callData = abi.encodeWithSignature("registerLidoWithdrawalRequests()");
+        console.log("registerLidoWithdrawalRequests data:");
+        console.logBytes(callData);
+
+        bytes memory proxyData = abi.encode(address(lidoARMImpl), callData);
+        console.log("proxy upgradeToAndCall encoded params:");
+        console.logBytes(proxyData);
+
+        govProposal.action(deployedContracts["LIDO_ARM"], "upgradeToAndCall(address,bytes)", proxyData);
+
+        _fork();
```

Comment:
This is a duplicate, and makes the `_fork()` get called twice, since `_fork()` is already called from the `AbstractDeployScript`.

Reply:
I copied this from the 004 script. I originally had problems with the tests without the `_fork()` call in `_buildGovernanceProposal`, but the tests now work locally without the extra `_fork()` call, so I've removed it.
Thanks